zipline/utils/sharedoc.py

```diff
@@ -11,17 +11,24 @@
     frequency : {'year_start', 'quarter_start', 'month_start', 'week_start'}
         A string indicating desired sampling dates:
 
-        'year_start'    -> first trading day of each year
-        'quarter_start' -> first trading day of January, April, July, October
-        'month_start'   -> first trading day of each month
-        'week_start'    -> first trading_day of each week
+        * 'year_start'    -> first trading day of each year
+        * 'quarter_start' -> first trading day of January, April, July, October
+        * 'month_start'   -> first trading day of each month
+        * 'week_start'    -> first trading_day of each week
     """
 )
 
 
-def pad_lines(prefix, s):
-    """Apply a prefix to each line in s."""
-    return '\n'.join(prefix + line for line in s.splitlines())
+def _gen_for_pad_lines_after_first(prefix, s):
+    lines = iter(s.splitlines())
+    yield next(lines)
+    for line in lines:
+        yield prefix + line
+
+
+def pad_lines_after_first(prefix, s):
+    """Apply a prefix to each line in s after the first."""
+    return '\n'.join(_gen_for_pad_lines_after_first(prefix, s))
```

Comment:
actually, are those not the same? the prefix isn't added to the first line in the join case.